app/javascript/src/Navigation/components/VerticalNav.jsx

```diff
@@ -20,19 +22,20 @@ type Section = Item & {
 type Props = {
   sections: Section[],
   activeSection: ?string,
-  activeItem: ?string
+  activeItem: ?string,
+  currentApi: ?Api
 }
 
-const VerticalNav = ({ sections, activeSection, activeItem }: Props) => (
+const VerticalNav = ({ sections, activeSection, activeItem, currentApi }: Props) => (
   <div className="pf-c-page__sidebar-body">
     <Nav id='mainmenu' theme='dark'>
-      <NavList>
+      <NavGroup title={currentApi.name ? currentApi.name : ''}>
```

Comment:
This is still going to throw, check the prop directly:
```suggestion
      <NavGroup title={currentApi ? currentApi.name : ''}>
```

However, I wonder if we're supposed to show an empty title when we're not in a Product or Backend page. It's going to show a weird empty space on top of the list.

Reply:
Perhaps we could put an empty NavGroup on top of the existing NavList and render it only when `currentApi` exists?
```jsx
{currentApi && <NavGroup title={currentApi.name} />}
<NavList>
   // ...
</NavList>
```

Reply:
NavGroup has to wrap around the sections, but I updated the code so that it will display the NavGroup wrapper if the currentAPI is present, and the NavList if its not.